File: Java_Course/src/App/MadLibs.java
package App;

import java.util.Scanner;

public class MadLibs {
	public static void madLibs(String [] args) {
		
		Scanner keyboardInput = new Scanner(System.in);
		
		System.out.print("Enter a color: ");
		String color = keyboardInput.nextLine();
		System.out.print("Enter a plural Noun: ");
		String pluralNoun = keyboardInput.nextLine();
		System.out.print("Enter a celebrity: ");
		String celebrity = keyboardInput.nextLine();
		
		System.out.println("Roses are " + color  + "\n"
							+ pluralNoun + " are blue\n"
							+ "I love " + celebrity + ".");	
		
		System.out.println();
	}
}

File: php_Course/www/madLibs.php
<!DOCTYPE html>
<html lang="en" dir="ltr">
  <head>
    <meta charset="utf-8">
    <title></title>
  </head>
  <body>

    <form action="madLibs.php" method="get">
      Enter a Color: <input type="text" name="color" value="red">
      <br>
      Enter a plural Noun: <input type="text" name="pluralNoun" value="Violets">
      <br>
      Enter a Celebrity: <input type="text" name="celebrity" value="you">
      <br>
      <input type="submit">
    </form>

    <br><br>

    <?php
      $color = $_GET["color"];
      $pluralNoun = $_GET["pluralNoun"];
      $celebrity = $_GET["celebrity"];

      echo "Roses are $color, <br>";
      echo "$pluralNoun are blue, <br>";
      echo "I love $celebrity. <br>";
     ?>

  </body>
</html>
